### PR TITLE
Unit test for '/info endpoint

### PR DESCRIPTION
**Pull request by JamesGardiner**
Adds an initial unit test covering expected behaviour from the `/info` endpoint.

### CODE REVIEW

**Comment by JamesGardiner** [comment]
[Survey Service changes for classifiers](https://trello.com/c/6HBmrwFN/6-survey-service-changes-for-classifiers)

**Comment by JamesGardiner** [comment]
any chance of a review @davidmort / @benjefferies?

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,13 +1,27 @@
 {
 	"ImportPath": "github.com/onsdigital/rm-survey-service",
-	"GoVersion": "go1.8",
+	"GoVersion": "go1.9",
 	"GodepVersion": "v79",
 	"Deps": [
+		{
```
Comment (benjefferies) [inline Godeps/Godeps.json]: Wonder if we need any "dev" deps as vendored deps

File: surveysvc_test.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/appleboy/gofight"
+	"github.com/buger/jsonparser"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInfo(t *testing.T) {
```
Comment (benjefferies) [inline surveysvc_test.go]: 👍

**Comment by benjefferies** [comment]
Wow thought there was more to the PR than there was. Was tricked by the vendor files

**Comment by andrewmil** [comment]
Is it normal practice to keep the vendor files in the repository?

It seems like that could get a bit excessive when projects get larger

**Pull request closed without merging**